test/parallel/test-url-pathtofileurl.js

```diff
@@ -23,6 +23,26 @@ const url = require('url');
   assert.ok(fileURL.includes('%25'));
 }
 
+{
+  if (isWindows) {
+    // UNC path: \\server\share\resource
+
+    // Missing server:
+    assert.throws(() => url.pathToFileURL('\\\\\\no-server'), {
```

Comment:
```suggestion
    assert.throws(() => url.pathToFileURL('\\\\\\\\no-server'), {
```

If the above comment makes sense.